styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt

```diff
@@ -133,7 +133,7 @@ interface Typo : StyleParams {
      * @param value extension function parameter for small media devices, recommended to use
      *           predefined values via [dev.fritz2.styling.theme.Theme.fonts] that offer the properties of [Fonts]
      */
-    fun fontFamily(value: Fonts.() -> Property) = property(fontSizeKey, Theme().fonts, value)
+    fun fontFamily(value: Fonts.() -> Property) = property(fontFamilyKey, Theme().fonts, value)
```

Comment:
Please correct the API doc example call too! ``body`` seems not to a an appropriate value there!

Reply:
no correction needed